hl7parser/hl7_data_types.py

```diff
@@ -115,6 +118,8 @@ def __nonzero__(self):
         elif len(self.input_fields) > 1:
             return True
 
+    def __bool__(self): # pragma: no cover
+        return self.__nonzero__()
```

Comment:
Das verstehe ich nicht. Warum brauchen wir das jetzt?

Reply:
Die Magic method für `bool(x)` war in Python 2 `__nonzero__` und ist jetzt `__bool__`